pkgs/by-name/wa/wayprompt/build.zig.zon.nix

```diff
@@ -0,0 +1,48 @@
+# generated by zon2nix (https://github.com/nix-community/zon2nix)
+
+{ linkFarm, fetchzip }:
+
+linkFarm "zig-packages" [
+  {
+    name = "12201bbf05e1fb73323a9b13d2599dc4bf82851e6d328d7523a1af5b861a87edf286";
+    path = fetchzip {
+      url = "https://git.sr.ht/~leon_plickat/zig-ini/archive/879c74a3a801d49fa34343aebd55a22f591899b3.tar.gz";
+      hash = "sha256-in7abjMSWtnmrri2tbTJW2WVL4ZLrKkTT8Bxy8njJT4=";
```

Comment:
Did you fill these in automatically?

I'm hitting this https://github.com/nix-community/zon2nix/issues/23

Reply:
> Did you fill these in automatically?
> 
> I'm hitting this [nix-community/zon2nix#23](https://github.com/nix-community/zon2nix/issues/23)

No, I was also hitting the same, that's why I didn't put an update script in automation. I manually filled those empty hashes.